### ISSUE TITLE
JsonSchema support

### ISSUE THREAD

**Issue by gpietro** [flipbyte/formik-json-schema#43]
### Is your feature request related to a problem? Please describe.

The library is named as formik-json-schema but it doesn't seem to support json schema format.

### Describe the solution you'd like

The solution would be to support the json schema definition for form rendering and validation.

**Comment by uragecz** [comment]
Facing some problem.... trying to make validation where i want to send regex or undefined value... but the problem is that json doesn't accept it... And wrapping it into string or doing some other magic doesn't work either..

**Comment by adjenks** [comment]
Yeah I really thought this library used actual "json-schema" (https://json-schema.org/) until I started reading the docs.
